blog/2024-07-31-open-planning-r24.12.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Tractus-X Open Planning R24.12
+description: The Eclipse Tractus-X Open Planning Days - Empowering the Catena-X Data Space
+slug: open-planning-r24-12
+date: 2024-04-16T11:00
+hide_table_of_contents: false
+authors:
+  - name: Stephan Bauer
+    title: Platform Domain Manager
+    url: https://github.com/stephanbcbauer
+    image_url: https://github.com/stephanbcbauer.png
+    email: stephan.bauer@catena-x.net
+---
+
+![eclipse tractus x logo](@site/static/img/Tractus-X_openplanning.png)
+
+Dear Tractus-X Community Members 🎉,
+
+we are excited to extend this invitation to you for our upcoming Tractus-X meetings. These meetings are a great opportunity for both business stakeholders and developers to contribute and collaborate on the future of Tractus-X.
+Please feel free to forward this post to anyone interested in contributing, whether in planning or development.
+
+## Open Meetings
+
+- **[Refinement Phase - Release 24.12](./open-planning-r24-12#refinement-phase---release-2412)**
```

Comment:
Not sure about this, but is the file reference needed?
Would expect, that just the `#headline-text` could be enough, but like said, actually not sure 😁

Reply:
Changed it, thanks for the hint. It still works. :)